test-harness/helpers.ts

```diff
@@ -1,3 +1,27 @@
+import { Dictionary } from '@stoplight/types/dist';
+import * as xmlDiff from 'diff-js-xml';
+import * as parser from 'fast-xml-parser';
+import { get, omit } from 'lodash';
+import * as typeIs from 'type-is';
+
+type Result = { body: string; headers: Dictionary<string, string> };
+
+const xmlValidator = {
+  test: (contentType: string, content: string) => {
+    const doesContentTypeMatch = !!typeIs.is(contentType, ['application/xml', 'application/*+xml', 'text/xml']);
+    const isContentXML = parser.validate(content) === true;
```

Comment:
In theory we could skip this check. Since we're the people writing the tests if we're not providing a correct xml payload we might want the test to just fail.

Reply:
@XVincentX, this is for when `get(output, ['header', 'content-type'], ''` returns `''`. Don't remember know exactly when it happens, but it must have since I added `const isContentXML = parser.validate(content) === true;` ;)